org.eclipse.jdt.core.tests.builder/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.jdt.core.tests.builder; singleton:=true
 Bundle-Version: 3.12.400.qualifier
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
+Fragment-Host: org.eclipse.jdt.core
```

Comment:
This ensures the tests shares the same classloader as jdt.core plugin, and the AbstractImageBuilder in jdt.core can create an instance of the MockCompiler in the tests.